presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraTokenSplitManager.java

```diff
@@ -25,9 +25,9 @@
 
 public class TestCassandraTokenSplitManager
 {
-    private static final int SPLIT_SIZE = 100;
+    private static final int SPLIT_SIZE = 256;
     private static final String KEYSPACE = "test_cassandra_token_split_manager_keyspace";
-    private static final int PARTITION_COUNT = 1000;
+    private static final int PARTITION_COUNT = 2048;
```

Comment:
Is this change necessary? Does this mean the `CassandraTokenSplitManager`has issues with non-power-of-two partition counts?